plugins/javadoc/src/main/resources/views/components/head.korte

```diff
@@ -5,8 +5,8 @@
     <link rel="stylesheet" type="text/css" href="{{ pathToRoot }}stylesheet.css" title="Style">
     <link rel="stylesheet" type="text/css" href="{{ pathToRoot }}dokka-javadoc-stylesheet.css" title="Style">
     <link rel="stylesheet" type="text/css" href="{{ pathToRoot }}jquery/jquery-ui.css" title="Style">
-    <script type="text/javascript" src="{{ pathToRoot }}jquery/jquery-3.3.1.js"></script>
-    <script type="text/javascript" src="{{ pathToRoot }}jquery/jquery-migrate-3.0.1.js"></script>
+    <script type="text/javascript" src="{{ pathToRoot }}jquery/jquery-3.5.0.js"></script>
```

Comment:
If you look inside `jquery-3.5.0.js`, you'll find that it's jquery 3.6.0. Can you please rename the file?

Reply:
It makes sense to add the values back in `KorteJavadocRenderer.kt` and use them here.